src/marqo/tensor_search/enums.py

```diff
@@ -78,6 +78,7 @@ class RequestType:
     DELETE = "DELETE"
     CREATE = "CREATE"
     PARTIAL_UPDATE = "PARTIAL_UPDATE"
+    EMBED = "EMBED"     # Request for embeddings of a given input content
```

Comment:
Do we need a separate throttling category? Note this is increasing concurrency. Maybe combine with search?

Reply:
Done.